keentools_facebuilder/blender_independent_packages/pykeentools_loader/install.py

```diff
@@ -314,18 +319,59 @@ def is_installed(force_recheck=False):
     return installation_status(force_recheck)[0]
 
 
-def _do_pkt_shadow_copy():
-    import tempfile
-    import shutil
+def _remove_dir(dir):
+    try:
+        shutil.rmtree(dir, ignore_errors=True)
+    except Exception as err:
+        logger = logging.getLogger(__name__)
+        logger.error('remove_dir: Cannot delete dir {}'.format(dir))
+        logger.error('Exception info: {}'.format(str(err)))
+
+
+def _get_all_pids_on_windows():
+    output = subprocess.run(['tasklist', '/NH', '/FO', 'CSV'],
+                            capture_output=True)  # windows specific code
```

Comment:
Remove comment. Maybe make assert if you need to make a point here

Reply:
Wrapped in try-except